package.json

```diff
@@ -2,17 +2,24 @@
   "name": "react-bare-bone-template",
   "version": "0.1.0",
   "private": true,
+  "type": "module",
   "scripts": {
     "build": "react-scripts build",
-    "spj": "npx sort-package-json@1.48 package.json",
     "dev": "react-scripts start",
-    "stylelint": "stylelint 'src/*.{js,jsx}'"
+    "dev-server": "nodemon server.js localhost 1337",
```

Comment:
does the localhost info go here, too?? I thought only in the server.js! ?

Reply:
I believe that will be autoamtic through the const port. But I am not sure

Reply:
that is what I thought! so to write it here is redundant? it doesnt break the code in any way?

Reply:
i think so, but no guarantee 😁